website/docs/r/lambda_alias.html.markdown

```diff
@@ -30,10 +35,17 @@ resource "aws_lambda_alias" "test_alias" {
 * `description` - (Optional) Description of the alias.
 * `function_name` - (Required) The function ARN of the Lambda function for which you want to create an alias.
 * `function_version` - (Required) Lambda function version for which you are creating the alias. Pattern: `(\$LATEST|[0-9]+)`.
+* `routing_config` - (Optional) The Lambda alias' route configuration settings. Fields documented below
+
+For **routing_config** the following attributes are supported:
+
+* `additional_version_weights` - (Optional) A map that defines the proportion of events that should be sent to different versions of a lambda function.
 
 ## Attributes Reference
 
 * `arn` - The Amazon Resource Name (ARN) identifying your Lambda function alias.
 
 [1]: http://docs.aws.amazon.com/lambda/latest/dg/welcome.html
 [2]: http://docs.aws.amazon.com/lambda/latest/dg/API_CreateAlias.html
+[3]: https://docs.aws.amazon.com/lambda/latest/dg/API_AliasRoutingConfiguration.html
```

Comment:
It looks like this documentation reference was added but never referenced above.